korgan/claim_filing_completeness.py

```
@@ -0,0 +1,113 @@
+"""
+
+from __future__ import annotations
+
+import re
+
+from korgan.claim_filing_accuracy import FILING_ACTION_PREFIX
+from korgan.legal_types import ClaimDraft, VerificationStatus
+
+_PLACEHOLDER_RE = re.compile(r"\[(?:ТРЕБУЕТ|НАҚТЫЛАУ|ТЕКСЕРУ)[^\]]*\]", re.IGNORECASE)
+_LEGAL_ENTITY_RE = re.compile(
+    r"(?:\bБИН\b|\bТОО\b|\bАО\b|\bНАО\b|\bРГП\b|\bРГКП\b|\bКГП\b|\bКГКП\b|\bКГУ\b|"
```

Ruff (0.16.1)

[warning] String contains ambiguous `Н` (CYRILLIC CAPITAL LETTER EN). Did you mean `H` (LATIN CAPITAL LETTER H)?

(RUF001)